php_phongo.h

```diff
@@ -117,15 +117,4 @@ zend_object_handlers* phongo_get_std_object_handlers(void);
 		phongo_throw_exception(PHONGO_ERROR_RUNTIME, "Accessing private constructor"); \
 	}
 
-#define PHONGO_DISABLED_WAKEUP(classname)                                                             \
-	static PHP_METHOD(classname, __wakeup)                                                            \
-	{                                                                                                 \
-		PHONGO_PARSE_PARAMETERS_NONE();                                                               \
-		phongo_throw_exception(PHONGO_ERROR_RUNTIME, "MongoDB\\Driver objects cannot be serialized"); \
-	}
-
-/* Shared function entries for disabling constructors and unserialize() */
-PHP_FUNCTION(MongoDB_disabled___construct);
```

Comment:
Noted that I added the macros in https://github.com/mongodb/mongo-php-driver/commit/6887226d34566cd772bbacb34e10fc7bd4000f57 but forgot to remove the shared functions below. Thanks for cleaning up my mess!